# Parameters for TUM RGB-D aka Freiburg3
camera:
  width: 640
  height: 480
  fx: 535.4
  fy: 539.2
  cx: 320.1
  cy: 247.6
  # Distortion model: 0=Pinhole, 1=Fisheye, 2=Brown, 3=Polynomial
  distortion_model: 0
  # For Brown: [k1, k2, p1, p2, k3]
  distortion_coeffs: [] 
